pkgs/by-name/hi/highs/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+}:
+
+lib.makeOverridable stdenv.mkDerivation rec {
+  pname = "highs";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "ERGO-Code";
+    repo = "HiGHS";
+    rev = "v${version}";
+    sha256 = "sha256-Wa5ivUJk0t58FhZD0zy0zxHHj4/p8e9WcxXwu5zenxI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  meta = with lib; {
+    description = "HiGHS: High performance serial and parallel linear solver";
```

```suggestion
    description = "High performance serial and parallel linear solver";
```
per nixpkgs manual, package name shouldn't be in desc